scripts/exgdas_global_marine_analysis_run.sh

```diff
@@ -39,8 +39,9 @@ function clean_yaml()
 
 ################################################################################
 # run 3DVAR FGAT
-clean_yaml var.yaml
-$APRUN_OCNANAL $JEDI_BIN/soca_var.x var.yaml
+cp var.yaml cleaned_var.yaml
+clean_yaml cleaned_var.yaml
+$APRUN_OCNANAL $JEDI_BIN/soca_var.x cleaned_var.yaml
 export err=$?; err_chk
```

Comment:
Seems fine, but I'd like to keep the `var.yaml` as the input to `var.x`. Can you instead copy `var.yaml` to `original_var.yaml` ... or something. You might have to copy  `original_var.yaml` in post.

Reply:
You mean you want the "cleaned" `var.yaml `for the `var.x`?

Reply:
yes